sql/core/benchmarks/JoinBenchmark-results.txt

```diff
@@ -2,74 +2,74 @@
 Join Benchmark
 ================================================================================================
 
-OpenJDK 64-Bit Server VM 1.8.0_222-b10 on Linux 3.10.0-862.3.2.el7.x86_64
+OpenJDK 64-Bit Server VM 1.8.0_232-8u232-b09-0ubuntu1~18.04.1-b09 on Linux 4.15.0-1044-aws
 Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
 Join w long:                              Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-Join w long wholestage off                         4685           4814         182          4.5         223.4       1.0X
-Join w long wholestage on                           440            524         102         47.7          21.0      10.7X
+Join w long wholestage off                         4531           4557          37          4.6         216.1       1.0X
+Join w long wholestage on                          1214           1310          95         17.3          57.9       3.7X
```

Comment:
@MaxGekk . For example, this case, this slowed down 3x times. It's a huge regression.

Reply:
Could you file a JIRA issue for this `JoinBenchmark`?

Reply:
Sure, I will do that tomorrow.

Reply:
Here is the JIRA: https://issues.apache.org/jira/browse/SPARK-30563